cupy/_math/misc.py

```diff
@@ -399,7 +399,27 @@ def nan_to_num(x, copy=True, nan=0.0, posinf=None, neginf=None):
     return _nan_to_num(x, nan, posinf, neginf, out=out)
 
 
-# TODO(okuta): Implement real_if_close
+def real_if_close(x, tol=100):
```

Comment:
```suggestion
def real_if_close(a, tol=100):
```
Please make the interface the same as `numpy.real_if_close`.